assets/css/main.css

```diff
@@ -3813,7 +3757,7 @@
                     #titleBar .toggle:before {
                         -moz-osx-font-smoothing: grayscale;
                         -webkit-font-smoothing: antialiased;
-                        font-family: FontAwesome;
```

Comment:
Why did you do this? Should be ` font-family: 'Font Awesome\ 5 Free';` if upgrading to V5 IIRC